README.md

```diff
@@ -104,57 +104,57 @@ Now you can sync your project and core features like serialization and deseriali
 **Groovy**
 
 ```groovy
-implementation "com.infinum.halley:halley-core:0.0.6"
+implementation "com.infinum.halley:halley-core:1.0.0"
```

Comment:
Having the same version in multiple places can be error prone (not updating it on one or more places).
We can improve it by defining var for version. Or gradle tasks that will update it in all places.

Not need to change this now, just keep this (small) improvement in mind.